apps/language_server/lib/language_server/providers/code_lens/spec.ex

```diff
@@ -0,0 +1,124 @@
+defmodule ElixirLS.LanguageServer.Providers.CodeLens.Spec do
```

Comment:
Can you change the name of this module (and filename) to `TypeSpec` instead of just `Spec`? I think that'll make it easier to understand at a glance. And just calling it "spec" reminds me a little too much of rspec (since I have a ruby background).

```suggestion
defmodule ElixirLS.LanguageServer.Providers.CodeLens.TypeSpec do
```